### PR TITLE
Bump to 2.1 ⬆️

### PR DESCRIPTION
**Pull request by chapulina**
* Merge from `ign-physics1`
* Update changelog
* Bump version

### CODE REVIEW

**Comment by azeey** [comment]
The [Ubuntu CI](https://build.osrfoundation.org/job/ignition_physics-ci-pr_any-ubuntu_auto-amd64/652/) is not finding sdformat9 because sdformat8 is installed.

File: Changelog.md [hunk 1]
```diff
@@ -36,6 +65,14 @@
 
 ### Ignition Physics 1.x.x (20XX-XX-XX)
 
+1. Fix collision issue with detachable joints
+    * [Pull request 31](https://github.com/ignitionrobotics/ign-physics/pull/31)
```
Comment (scpeters) [inline Changelog.md:69]: I think this should be added to the 2.1.0 section as well

**Comment by scpeters** [comment]
> The Ubuntu CI is not finding sdformat9 because sdformat8 is installed.

https://github.com/ignition-tooling/release-tools/pull/194

File: Changelog.md (hunk 1 above)
Comment (scpeters) [reply Changelog.md:72]: c28c24f

**Pull request merged**